src/macad_gym/envs/follow_leading_vehicle.py

```diff
@@ -0,0 +1,139 @@
+"""
+Author: Morphlng
+Date: 2023-01-28 18:33:26
+LastEditTime: 2023-01-28 18:33:28
+LastEditors: Morphlng
+Description: Example of creating a custom environment, also a demonstration of how to use the manual_control.
+FilePath: \src\macad_gym\envs\follow_leading_vehicle.py
+"""
+
+import random
+from macad_gym.envs import MultiCarlaEnv
+
+"""
```

Comment:
Nice example!
It will be great if you can also add a  [Wiki page here](https://github.com/praveen-palanisamy/macad-gym/wiki) to show how one could import/convert existing Carla/scenario_runner scenarios into RL-enabled MACAD-Gym Envs.